openapi/p4pa-pu-bff.openapi.yaml

```diff
@@ -706,6 +745,53 @@ components:
         number:
           type: integer
           format: int64
+    ReceiptDetailDTO:
+      type: object
+      required:
+        - receiptId
+        - paymentAmountCents
+        - remittanceInformation
+        - debtPositionDescription
+        - debtor
+        - payer
```

Comment:
let's remove from required fields, actually each payment process is an anonymous operation (no login), thus it will never be filled